packages/customer-account-ui-extensions-react/src/hooks/i18n.ts

```diff
@@ -0,0 +1,15 @@
+import {
+  RenderExtensionPoint,
+  I18n,
+} from '@shopify/customer-account-ui-extensions';
+import {useApi} from './api';
+
+/**
+ Returns utilities for translating content and formatting values
+    according to the current localization of the user.
+ */
+export function useI18n<
+  ID extends RenderExtensionPoint = RenderExtensionPoint,
+>(): I18n {
+  return useApi<ID>().i18n;
```

Comment:
As you yourself mentioned, in checkout they have more fine-grained hooks (e.g. translate) that  do some more work here. I think we should check why and potentially do the same, unless there's a good reason not to. I'd be ok if this happens in a follow-up PR if that's better for you. 

https://github.com/Shopify/ui-extensions/blob/633503bc8d64a3bea7b5b9b2537b3d91b7e3c7a6/packages/checkout-ui-extensions-react/src/hooks/translate.ts#L13